posthog/batch_exports/http.py

```diff
@@ -87,31 +91,15 @@ def list(self, request: request.Request, *args, **kwargs) -> response.Response:
         if not isinstance(request.user, User) or request.user.current_team is None:
             raise NotAuthenticated()
 
-        after = self.request.query_params.get("after", None)
+        after = self.request.query_params.get("after", "-7d")
         before = self.request.query_params.get("before", None)
-        date_range = None
-        if after is not None and before is not None:
-            after_datetime = validate_date_input(after)
-            before_datetime = validate_date_input(before)
-            date_range = (after_datetime, before_datetime)
-
-        runs = self.get_queryset(date_range=date_range)
-        limit = self.request.query_params.get("limit", None)
-        if limit is not None:
-            try:
-                limit = int(limit)
-            except (TypeError, ValueError):
-                raise ValidationError(f"Invalid value for 'limit' parameter: '{limit}'")
-
-            runs = runs[:limit]
-
-        page = self.paginate_queryset(runs)
-        if page is not None:
-            serializer = self.get_serializer(page, many=True)
-            return self.get_paginated_response(serializer.data)
-
-        serializer = self.get_serializer(runs, many=True)
-        return response.Response(serializer.data)
+        after_datetime = relative_date_parse(after)
+        before_datetime = relative_date_parse(before) if before else now()
+        date_range = (after_datetime, before_datetime)
+
+        page = self.paginate_queryset(self.get_queryset(date_range=date_range))
+        serializer = self.get_serializer(page, many=True)
+        return self.get_paginated_response(serializer.data)
```

Comment:
The reset tests are failing as the method we use to list runs doesn't appear to have a `count` key anymore. However, I don't see any changes here that could have dropped that field 🤔

Reply:
ah yeah I missed those tests. Will check it out now